src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoTreeNodes.cs

```diff
@@ -921,6 +921,20 @@ public override IEnumerable<INodeFilter> Filters
             }
         }
 
+        public override IEnumerable<NodeSmoProperty> SmoProperties
+        {
+            get
+            {
+                var properties = new List<NodeSmoProperty>();
+                properties.Add(new NodeSmoProperty
+                {
+                   Name = "LedgerViewType",
+                   ValidFor = ValidForFlag.Sql2022|ValidForFlag.AzureV12
```

Comment:
>    ValidFor = ValidForFlag.Sql2022|ValidForFlag.AzureV12 [](http://example.com/codeflow?start=15&length=57)

is MI the same as Sql2022?

Reply:
I think so, since SQL Server version/build of Managed Instance is always the same as latest stable SQL Server database engine.